dev-tools/release/smoketest_rc.sh

```diff
@@ -0,0 +1,220 @@
+#!/bin/bash
+
+
+
+set -e
+
+usage() {
+    echo "Usage: smoketest-rc.sh -v version_number -m maven_coordinates"
+    echo "  -v   The #.#.#.RC# version number"
+    echo "  -m   The maven url to download artifacts from"
+    exit 1
+}
+
+checkCommand() {
+    COMMAND=$1
+    if ! [[ -x "$(command -v $COMMAND)" ]]; then
+        echo "$COMMAND must be installed"
+        exit 1
+    fi
+}
+
+FULL_VERSION=""
+MAVEN_URL=""
+
+while getopts ":v:m:" opt; do
+  case ${opt} in
+    v )
+      FULL_VERSION=$OPTARG
+      ;;
+    m )
+      MAVEN_URL=$OPTARG
+      ;;
+    \? )
+      usage
+      ;;
+  esac
+done
+
+if [[ ${FULL_VERSION} == "" ]] || [[ ${MAVEN_URL} == "" ]]; then
+    usage
+fi
+
+if [[ $FULL_VERSION =~ ^([0-9]+\.[0-9]+\.[0-9]+)\.(RC[0-9]+)$ ]]; then
+    VERSION=${BASH_REMATCH[1]}
+else
+    echo "Malformed version number ${FULL_VERSION}. Example valid version: 1.9.0.RC1"
+    exit 1
+fi
+
+VERSION_MM=${VERSION%.*}
+
+checkCommand gpg
+checkCommand cmake
+#checkCommand svn
+checkCommand doxygen
+
+
+echo ""
+echo "============================================================"
+echo "Checking java..."
+echo "============================================================"
+[ -z "$JAVA_HOME" ] && JAVA=java || JAVA=$JAVA_HOME/bin/java
+if ! $JAVA -XshowSettings:properties -version 2>&1 | grep 'java.specification.version = 1.8' ; then
+  echo "Please set JAVA_HOME to use JDK 8 to compile Geode for release"
```

Comment:
for validation, any vendor is fine, just needs to be JDK8.  and can remove the words "for release"

Reply:
done